templates/iot/common/030-Deployment-iot-interior-interconnect.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
```

Comment:
For all the deployments: I think it is ok to leave this for the moment, but we must extend the operator to take care of this.